pkg/apis/loadbalancer.harvesterhci.io/v1alpha1/type.go

```diff
@@ -11,8 +11,8 @@ import (
 // +kubebuilder:resource:shortName=lb;lbs,scope=Namespaced
 // +kubebuilder:printcolumn:name="DESCRIPTION",type=string,JSONPath=`.spec.description`
 // +kubebuilder:printcolumn:name="TYPE",type=string,JSONPath=`.spec.type`
-// +kubebuilder:printcolumn:name="INTERNALADDR",type=string,JSONPath=`.status.internalAddress`
-// +kubebuilder:printcolumn:name="EXTERNALADDR",type=string,JSONPath=`.status.externalAddress`
+// +kubebuilder:printcolumn:name="INTERNAL-ADDRESS",type=string,JSONPath=`.status.internalAddress`
```

Comment:
Refer to the service and use - to separate the words. You can try kubectl get svc -o wide.